src/app/pages/salidas/form-elementos-seleccionados/form-elementos-seleccionados.component.ts

```diff
@@ -173,20 +172,20 @@ export class FormElementosSeleccionadosComponent implements OnInit {
   private validarTercero(): ValidatorFn {
     return (control: AbstractControl): ValidationErrors | null => {
       const valor = control.value;
-      const checkStringLength = typeof (valor) === 'string' && valor.length < 4 && valor !== '' ? true : false;
-      const checkInvalidString = typeof (valor) === 'string' && valor !== '' ? true : false;
-      const checkInvalidTercero = typeof (valor) === 'object' && !valor.Tercero ? true : false;
+      const checkStringLength = typeof (valor) === 'string' && valor.length < 4 && valor !== '';
+      const checkInvalidString = typeof (valor) === 'string' && valor !== '';
+      const checkInvalidTercero = typeof (valor) === 'object' && !valor.Tercero;
       return checkStringLength ? { errorLongitudMinima: true } :
-        checkInvalidString || checkInvalidTercero ? { terceroNoValido: true } : null;
+        (checkInvalidString || checkInvalidTercero) ? { terceroNoValido: true } : null;
```

Comment:
Para confirmar/finalizar... toda esta línea sería "el `else` de la primer terna", si?
```suggestion
        ((checkInvalidString || checkInvalidTercero) ? { terceroNoValido: true } : null);
```

Reply:
Sería más como un else if, el else sería cuando no hay ningún error